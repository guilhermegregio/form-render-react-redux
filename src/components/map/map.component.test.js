import React from 'react';
import ReactDOM from 'react-dom';

import Map from './map.component';

it('default render map component', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Map className="map" />, div);

    expect(div).toMatchSnapshot();
});
