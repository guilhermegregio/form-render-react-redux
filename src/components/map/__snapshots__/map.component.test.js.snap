// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`default render map component 1`] = `
<div>
  <div
    data-reactroot=""
    class="map-component group map"
  />
</div>
`;
